conn.go

```diff
@@ -3284,16 +3293,23 @@ func (cn *conn) Conn_authenticate(o values) (status bool, err error) {
 		saltStr := string(salt)
 		elog.Debugf(chopPath(funName()), "Salt value  %s\n", saltStr)
 		w := cn.writeBuf('p')
+		var sFinal string
+		const MinJWTTokenLen = 128 //Minimum length of the JWT token
 
+		if len(o["password"]) > MinJWTTokenLen && isJWT(o["password"]) {
+			sFinal = o["password"]
+			elog.Debugln(chopPath(funName()), "Password is a JWT token")
+		} else {
 		digest := sha256.New()
 		digest.Write([]byte(saltStr))
 		digest.Write([]byte(o["password"]))
 		sha256Sum := digest.Sum(nil)
 		elog.Debugln(chopPath(funName()), "sha256 sum ", sha256Sum)
 
 		sEnc := b64.StdEncoding.EncodeToString(sha256Sum) //Base 64 bit encoding (24 bytes)
-		sFinal := strings.TrimRight(sEnc, "=")            //remove trailing '=' characters
+		sFinal = strings.TrimRight(sEnc, "=")            //remove trailing '=' characters
```

Comment:
@Digvijay1999 can u please correct this syntax ?

Reply:
I have removed ":=" and changed to assignment operator because, I have moved the declaration of sFinal at the start of the function.
var sFinal string